frame/election-provider-multi-phase/test-staking-e2e/src/mock.rs

```diff
@@ -124,6 +124,10 @@ impl pallet_balances::Config for Runtime {
 	type DustRemoval = ();
 	type ExistentialDeposit = ExistentialDeposit;
 	type AccountStore = System;
+	type MaxHolds = ();
```

Comment:
how was this not caught in the CI? is this being compiled as a part of CI? 🙈

Reply:
Should be good now!